script/Ciel/Day 5/05_01_CIEL05_4D.txt

```diff
@@ -86,83 +98,116 @@ I mean, even with a gag order, there's going to be leaks, right?"
+"Oh it'll work.
+ %{i}Hooo%{/i}, man we lucked out that this went down in a public park.
+ Perfect excuse for why there were a bunch a unrelated people in the same place."
 }
 [sha:260e6fea6049d4e3def888f6142f9558abb09066]{
 -- Page 795, Offset 13008.
 -- 「理由さえあればあとは力仕事だわな。何の為にうちのジジイが権力バラまいてると思ってやがる。押さえるのは<上|トップ>だけでいいんだよ。下っ端や真ん中が不審がっても関係ねえ。最後にまとめる人間が握りつぶせばそれでおしまいだ」
-"As long as we have a pretext, the rest is just a matter of manpower. Why do you think my old man's handing out power for? The only people we have to keep in check are the ones on top. It doesn't matter if the underlings or those in the middle get suspicious. As long as the people in charge don't let anything through, it'll go nowhere."
+// Reading 散播く as _wielding_ political power rather than _disseminate_ (i.e, 散播き政治)
+// since that seems to make more sense in ctx
+"So long as we have a cover story, the rest is just a matter of legwork.
+ Why d'ya think my old man's been leaning on people left and right?
+ All you gotta do is put pressure on the guys at the top.
+ Doesn't make a fat bit of difference if the small fry or middle managers get suspicious.
+ If the people in charge crush 'em, that'll be the end of it."
 }
 [sha:6ea15e6faee1e6139fd1dc7d3d3a075c1d9a1735]{
 -- Page 796, Offset 13009. Mods: @k, @e.
 -- 「……はあ、そりゃあ辛い。そっかー、そういう風に闇に葬られてきたのかー。
-"...Haa, that's harsh. I see, so that's how it's going to be hushed up.
+"%{i}Sigh%{/i}... no mercy huh.
+ I see, so that's how you plan on hushing it all up.
 }
 [sha:68e29f8b315c8e3e2ce21da1695e602aecf16ecb]{
 -- Page 796, Offset 13010. Glued. Mods: @x.
 -- 坊ちゃん、その歳でエリートコースまっしぐらッスね。家柄的な話プラス、精神的な話で。なんかこう、もっと年相応の遊び心とかないんですか？」
- Young master, at your age, you're going on the fast track at full speed. Speaking of your pedigree and mentality. I mean, don't you feel the desire to act a bit more like the other kids your age and go have some fun?"
+// I think canon is that he's 'adopted', right? So lineage/pedigree for 家柄
+// probably isn't the best, it's also something that can't be actively
+// improved, so it's not like he can be dedicating himself to it instead of
+// more 年相応の遊び. Rendering as social standing instead.
+// Mario grindset
+ You're still young but you're already gunning it for the fast track aren't you, young master?
+ Both in terms of your social status and your mindset.
+ I mean, don't you ever want to act your age and play around once in a while?"
 }
 [sha:6b5a46592fca4924d9eda81d955c113f1bc5c10d]{
 -- Page 797, Offset 13011.
 -- 「あるかンなもん。ガキってのは基本つまらねえもんだって忘れたのか30歳？　オラ、無駄話してねえで監督してきやがれ。ヤバめな証拠は隠滅しとけよ」
 // He says 30歳, thirty year-old. Just imagine that I thought about "30 year old boomer" for a second haha
-"No fucking way. Have you forgotten that brats are all dull, you 30-year-old-geezer? Quit it with your nonsense and go supervise. And make sure to destroy any dangerous evidence."
+"Absofucking%{i}lutely%{/i} not.
```

Comment:
Abso-fucking-%{i}lutely%{/i} not.

might look completely fucked with italics but /shrug